site/management.md

```diff
@@ -288,25 +288,31 @@ rabbitmqctl set_user_tags full_access administrator
 RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
 not automatically redirect users to authenticate
 against the OAuth 2 server, this must be configured separately. Currently,
-RabbitMQ has been tested against the following Authorization servers:
+**Authorization code flow with PKCE** has been tested against the following Authorization servers:
 
 * [UAA](https://github.com/cloudfoundry/uaa)
 * [Keycloak](https://www.keycloak.org/)
 * [Auth0](https://auth0.com/)
 * [Azure](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/auth-oauth2)
 
+**IMPORTANT**: In OAuth 2.0 terms, RabbitMQ management UI is a **public app** which
+means it cannot securely store credentials such as `client_secret`. Your authorization server must allow you
+to configure the oauth client as **public app**. This means that RabbitMQ does not need to present a client_secret
+when authenticating/authorizing users. If your authorization server only supports **confidential app**, or
+simply it always requires `client_secret`, only then you have to configure one via the `oauth_client_secret` setting.
+
 To redirect users to the UAA server to authenticate, use the following configuration:
 
 <pre class="lang-ini">
-management.enable_uaa = true
+management.enable_uaa = true  
 management.oauth_enabled = true
 management.oauth_client_id = rabbit_user_client
-management.oauth_client_secret = rabbit_user_client_secret
 management.oauth_provider_url = https://my-uaa-server-host:8443/uaa
 </pre>
 
 > IMPORTANT: Since RabbitMQ 3.10, RabbitMQ uses `authorization_code` grant type. `implicit` flow has been
 deprecated.
+> IMPORTANT: management.oauth_client_secret is an optional setting. It is only required when your authorization server requires it
 
```

Comment:
management.oauth_client_secret needs to be tagged as code in Markdown as it is a code entity correct? So: 

IMPORTANT: The `management.oauth_client_secret` is an optional setting, which is only required when your authorization server requires it.

Reply:
indeed